osu.Game/Screens/Play/HUD/ArgonCounterTextComponent.cs

```diff
@@ -83,6 +75,8 @@ public ArgonCounterTextComponent(Anchor anchor, LocalisableString? label = null)
                     }
                 }
             };
+
+            RequiredDisplayDigits.BindValueChanged(digits => wireframesPart.Text = new string('#', digits.NewValue));
```

Comment:
This is not a correct replacement for the previous logic. The intent of the previous logic was that the number of wireframe digits should be _at least_ `RequiredDisplayDigits`, but if there are more digits currently displayed, wireframe digits should be _added_ to compensate.

This is visible when reaching 1M score in standardised mode. The wireframe will not be displayed behind the leading 1 anymore.

![osu_2024-01-08_10-04-31](https://github.com/ppy/osu/assets/20418176/94218362-049c-44a1-9c37-95b9c0470f51)

Reply:
Funny because I read the original logic at least ten times and this wasn't obvious. Says something about how convoluted it was in the first place.

Reply:
Have fixed this in 04fd83037b973c36cb2014219a59ab4ba5bd9b48.

Reply:
Fixing this locally to `ArgonScoreCounter` still changes behaviour. On `master` the wireframe is visible e.g. behind the combo counter which is not an `ArgonScoreCounter`. On this PR, there is no wireframe at all behind the combo counter (because it doesn't explicitly set `RequiredDisplayDigits`).

Reply:
I did wonder to myself while testing "did the combo always have no wireframe? i guess that was a design choice.."